documentation-site/components/posts.js

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+/* global process */
+
+import React from 'react';
+import {Block} from 'baseui/block';
+import logo from '../images/base-web.svg';
+import {Card, StyledBody, StyledAction} from 'baseui/card';
+import {Button, KIND} from 'baseui/button';
+import {styled} from 'baseui';
+import Layout from './layout';
+import Markdown from './markdown-elements';
+import posts from '../posts';
+
+const MetaData = styled('h2', ({$theme}) => ({
+  color: $theme.colors.mono700,
+  fontFamily: $theme.typography.font100.fontFamily,
+  fontSize: $theme.sizing.scale500,
+  lineHeight: $theme.sizing.scale600,
+  margin: 0,
+  fontWeight: 300,
+}));
+
+const Index = () => {
+  return (
+    <Block display="flex" flexWrap="wrap">
+      {posts && !posts.length && <h1>No posts to display</h1>}
+      {posts &&
+        posts.length > 0 &&
+        posts.map((p, i) => {
+          return (
+            <Card
+              key={`post--${i}`}
+              href={p.path}
+              title={p.title}
+              headerImage={p.coverImage}
+              overrides={{
+                Root: {
+                  style: {
+                    boxSizing: 'border-box',
+                    width: '300px',
+                    margin: '0 10px 10px 0',
```

Comment:
to make sure it is future proof, can you please use long hand props? https://www.styletron.org/concepts/#shorthand-and-longhand-properties